getting_started/administrators.adoc

```diff
@@ -67,43 +67,23 @@ already listening on those ports you must stop that service before launching
 the {product-title} container.
 ====
 
-*Installing and Starting an All-in-One Server*
+*Starting an All-in-One Server*
 
-. Launch the server in a container:
+. Launch the server:
 +
 ----
-$ sudo docker run -d --name "origin" \
-        --privileged --pid=host --net=host \
-        -v /:/rootfs:ro -v /var/run:/var/run:rw -v /sys:/sys -v /sys/fs/cgroup:/sys/fs/cgroup:rw \
-        -v /var/lib/docker:/var/lib/docker:rw \
-        -v /var/lib/origin/openshift.local.volumes:/var/lib/origin/openshift.local.volumes:rslave \# <1>
-        openshift/origin start
+$ oc cluster up
 ----
-<1> `rslave` only works if the Docker version is 1.10 or later and a Red Hat distribution.
-+
-[NOTE]
-====
-If you want to use xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[{product-title}'s
-aggregated logging], you must add `-v /var/log:/var/log` to the `docker` command
-line. The *origin* container must have access to the host's
-*_var/log/containers/_* and *_/var/log/messages_*.
-====
 +
 This command:
 +
-- starts {product-title} listening on all interfaces on your host (*0.0.0.0:8443*),
-- starts the web console listening on all interfaces at `/console` (*0.0.0.0:8443*),
-- launches an [sysitem]#etcd# server to store persistent data, and
-- launches the Kubernetes system components.
-
-. After the container is started, you can open a console inside the container:
-+
-----
-$ sudo docker exec -it origin bash
-----
+- starts {product-title} listening on local interface on your host (*127.0.0.1:8443*),
+- starts the web console listening on all interfaces at `/console` (*127.0.0.1:8443*),
+- launches the Kubernetes system components,
+- and provisions registry, router, initial templates, and a default project.
 
-If you delete the container, any configuration or stored application definitions
-will also be removed.
+If you want to learn more, use `oc cluster --help` or `oc cluster up --help` to
+get more information about all the options it provides.
```

Comment:
Think the phrasing should be reversed or just omit the `oc cluster up --help` altogether and keep things stupid-simple:
```
If you want to learn more, use `oc cluster --help` to get more information about all the options it provides.
```